lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -489,6 +491,7 @@ def _refresh(
         channel: Optional[str] = "",
         cohort: Optional[str] = "",
         revision: Optional[str] = None,
+        devmode: Optional[bool] = False,
```

Comment:
I don't think we need an optional here, just `devmode: bool = False` (with a default of False) is enough. None doesn't have a special meaning.

Reply:
Done